### DISCUSSION TITLE
Access Control for custom extension module

### DISCUSSION THREAD

**Discussion by phongXEM** [directus/directus#22408] [Feature Requests]
### Summary

Access Control for custom extension module.
Only roles allowed can view extension module.

### Basic Example

Now anyone can go to this link to watch.  I want to hide this icon link.
<img width="880" alt="Screenshot at May 07 08-58-25" src="https://github.com/directus/directus/assets/141018948/c1e52591-9ad4-4ee6-8b0e-3a3dea5f9e12">

### Motivation

Access Control modules.

### Detailed Design

Access Control modules.

### Requirements List

Now anyone can go to this link to watch.  I want to hide this icon link.
<img width="880" alt="Screenshot at May 07 08-58-25" src="https://github.com/directus/directus/assets/141018948/c1e52591-9ad4-4ee6-8b0e-3a3dea5f9e12">

### Drawbacks

Now anyone can go to this link to watch.  I want to hide this icon link.
<img width="880" alt="Screenshot at May 07 08-58-25" src="https://github.com/directus/directus/assets/141018948/c1e52591-9ad4-4ee6-8b0e-3a3dea5f9e12">

### Alternatives

Now anyone can go to this link to watch.  I want to hide this icon link.
<img width="880" alt="Screenshot at May 07 08-58-25" src="https://github.com/directus/directus/assets/141018948/c1e52591-9ad4-4ee6-8b0e-3a3dea5f9e12">

### Adoption Strategy

Now anyone can go to this link to watch.  I want to hide this icon link.
<img width="880" alt="Screenshot at May 07 08-58-25" src="https://github.com/directus/directus/assets/141018948/c1e52591-9ad4-4ee6-8b0e-3a3dea5f9e12">

**Comment by hanneskuettner** [comment]
Heya, looks like this is a duplicate of #3211 and I'll proceed to close this. You can upvote and follow along in the linked discussion.
